test/integration/auth/svcaccttoken_test.go

```diff
@@ -435,6 +435,56 @@ func TestServiceAccountTokenCreate(t *testing.T) {
 		}
 	})
 
+	t.Run("extended expiration extended does not apply for other audiences", func(t *testing.T) {
+		var requestExp int64 = 60*60 + 7
```

Comment:
hoist this to a private package constant and use it in this test and "expiration extended token", so we're sure the same expiration triggers an extended token, depending on the audiences